README.rst

```diff
@@ -235,6 +240,21 @@ you can enable application default credentials using the sdk:
 
     $ gcloud auth application-default login
 
+
+
+Using oauth tokens you can authorize by doing this:
+
+.. code:: sh
+
+    $ export CLOUDSDK_AUTH_ACCESS_TOKEN=<your access token>
+
+If you are already logged in 
+
+.. code:: sh
+
+    $ export CLOUDSDK_AUTH_ACCESS_TOKEN=$(gcloud auth print-access-token)
+
```

Comment:
Shouldn't you mention that `CLOUDSDK_AUTH_ACCESS_TOKEN` can also point to a file?

Also, does the Google Cloud SDK also support this double usage of `CLOUDSDK_AUTH_ACCESS_TOKEN`? If not, why support it here (but don't document it)?

Reply:
Yes, of course, i agree - i will remove the possibility to add it as a reference to a file.